### ISSUE TITLE
[Accessibility] Talkback does not announces/read string/description label over Text

### ISSUE THREAD

**Issue by paragag** [facebook/react-native#25442]
React Native version: 0.55.4

## Steps To Reproduce

1. Create a react native layout where a text is used inside a view/TouchableHighlight . Pasted below for references
2. without accessibility, normal operation this works out fine.. i see both read and "Details:- Not spoken" labels. Also clicking/tapping on "Details:- Not.." spoken button works as expected. 
3. But when i turn on accessibility, talkback does not  navigate to the "Details:- ..", nor does it speaks/reads it. Both the text are visible though. Even tapping on "Details:- Not" does not takes the controls there. 

Describe what you expected to happen:
while swapping left right, the navigation/highlight should go to "Details:- " button/text, so that user can enter it and see the details. 

```
<View>
      <View >
        <View accessible={true} accessibilityLabel="READ">
          <Text style={styles.sendActionTextLabel}>"READ"</Text >
        </View>
        <TouchableHighlight onPress={this.MyPage.bind(this)} underlayColor='transparent' accessibilityTraits={'button'} accessibilityComponentType={'button'} >
          <Text accessible={true} style={styles.discoverButtonTextLabel} accessibilityLabel="Double tap to get details">"Details : - NOT SPOKEN BY ACCESSIBILITY or accessible either by swipes"</Text >
        </TouchableHighlight >
      </View>
  </View>
```

**Comment by kelset** [comment]
Is this still the case for version 0.59? What about 0.60-rc3?

If it doesn't happen on those versions anymore we can close this.
